odoo/models.py

```diff
@@ -6160,22 +6160,35 @@ def _get_placeholder_filename(self, field=None):
         return 'placeholder.png'
 
     def _populate_factories(self):
-        """ Return a list of pairs ``(field_name, factory)``, ``factory`` is a function that creates
-        a generator of values (dict of field values).  Its API suggested by::
+        """ Return a list of pairs ``(field_name, factory)``.
```

Comment:
That''s redundant with the `returns`. What does it actually do, semantically? e.g. "Generates a factory for each fied of the model" or something.